### PR TITLE
Add some automation for persistent damage recovery DCs

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/equipment-effects/effect-ichthyosis-mutagen.json [hunk 1]
```diff
@@ -24,6 +24,16 @@
...
+            },
+            {
+                "itemType": "condition",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "predicate": [
+                    "item:damage:type:bleed"
+                ],
+                "property": "pd-recovery-dc",
+                "value": 1
```
Comment (TikaelSol) [inline packs/equipment-effects/effect-ichthyosis-mutagen.json:36]: Does this auto succeed on a nat 1? The system will still downgrade the success to a failure there I think.,

Comment (stwlam) [reply packs/equipment-effects/effect-ichthyosis-mutagen.json:36]: This would be done better with a DoS adjustment

Comment (TikaelSol) [reply packs/equipment-effects/effect-ichthyosis-mutagen.json:36]: I agree

Comment (simonward) [reply packs/equipment-effects/effect-ichthyosis-mutagen.json]: That's a good idea!

It looks like the damage type isn't being passed to the "flat-check" selector so I've reverted the changes to this file, and I'll submit a separate PR.

**Pull request merged**